### DISCUSSION TITLE
Does Alpha or Theta apply to 2 Axis Positioner?

### DISCUSSION THREAD

**Discussion by mjnewsum** [visose/Robots#233] [Q&A] [answered]
I am setting up a 2 axis positioner and I noticed that the location/orientation of the 3d model seems to be the only thing determining the rotational axis of the positioner joints. 
I tried to set the α and θ variables in the Revolute Joints, but they seem to have no affect.

Am I missing something or are Theta and Alpha ignored for the Positioner Mechanism? 

Ideally, I would like to set the first joint of the positioner to be at the XY origin plane in rhino with Z being the axis of rotation, and have the DH parameters determine the offsets and rotations to the following joint's axis of rotation. This is how the RobotArm works, so I am surprised it isn't the same for the Positioner.

**Comment by visose** [comment, marked as answer]
Yes, if you're using the positioner mechanism it's not generalized to use all DH parameters, but this should be simple to add since the plugin already has general FK solvers for both DH and modified DH params.

Feel free to clone repo and create an external mechanism class that does this.

**Comment by mjnewsum** [comment]
Thank you @visose!

I just wanted to make sure this is how the mechanism is designed to work. 
I will try making my own external mechanism that fits my requirements.
